content/getting-started/white-label-apps.md

```diff
@@ -58,7 +58,7 @@ You should try any scripts on your local machine to test things like authenticat
 
 If you are building many different versions, then instead of creating a long codemagic.yaml file with tens of workflows, you can have a single workflow and use the Codemagic API to pass environment variables to change build parameters.
 
-Users can start new builds using [Codemagic REST API](../rest-api/codemagic-rest-api.md). All data is sent and received as JSON. The REST API supports an optional parameter for environment variables. 
+Users can start new builds using [Codemagic REST API](https://docs.codemagic.io/rest-api/applications/). All data is sent and received as JSON. The REST API supports an optional parameter for environment variables. 
```

Comment:
Can you use a relative link instead? This should look something like this: /rest-api/applications/

Reply:
Relative path gives an empty page.

Reply:
Did you test it locally? Works for me.

Reply:
The problem is that the relative link has `.md` on the end. It should be `../rest-api/codemagic-rest-api/`

Reply:
Changed it!